queue_job/fields.py

```diff
@@ -40,7 +40,7 @@ class JobSerialized(fields.Field):
     def __init__(self, string=fields.Default, base_type=fields.Default, **kwargs):
         super().__init__(string=string, _base_type=base_type, **kwargs)
 
-    def _setup_attrs(self, model, name):
+    def _setup_attrs(self, model, name):  # pylint: disable=W8110
```

Comment:
nit: the human-readable code is preferable

Reply:
Agree, I changed them